### DISCUSSION TITLE
Problem with delay in new version ?

### DISCUSSION THREAD

**Discussion by lezed06** [AutomaApp/automa#372] [Q&A]
Hello, I think there is a problem with the delay fonction in the new version.
I made workflow who doing well the job (with a delay of 10000 betwen each action), but with the new version the delays are not respected.
Thanks

**Comment by simpleson2022** [comment]
Yes, my delay in the new version will fail if it exceeds 10,000ms, and using Javascript as a delay does not work either, do you have a solution?

**Reply by Kholid060** [reply]
Can you show the workflow log or send the workflow json file to reproduce this issue? Because everything is working even when I set the delay to 60,000ms

![image](https://user-images.githubusercontent.com/22908993/203513570-60e6fc8f-6ca0-4ef9-acce-680dbe7a6c60.png)

**Comment by simpleson2022** [comment]
![image](https://user-images.githubusercontent.com/117051726/203731160-a69570ec-4e56-40ae-a110-f2edca119c67.png)
![image](https://user-images.githubusercontent.com/117051726/203731190-922aed28-d21a-4003-9508-b541f2eb5605.png)

**Comment by simpleson2022** [comment]
I use firefox, automa version as below:
![image](https://user-images.githubusercontent.com/117051726/203731399-f8878522-d3dd-4477-89c9-18db9289336e.png)

**Comment by simpleson2022** [comment]
I already know where the problem is, it is normal to switch to Google browser.
thank you.
